lib/setup/setupSetup.js

```diff
@@ -272,8 +272,19 @@ function Setup(options) {
         }
     }*/
 
+    /**
+     * Asks the user if he wants to migrate objects if it makes sense and performs migration according to input
+     *
+     * @param {object} newConfig - updated config
+     * @param {object} oldConfig - previous config
+     * @param {object} rl - readline object
+     * @param {function(number=):void} callback - callback function
```

Comment:
I hate that WebStorm doesn't support the arrow style syntax. This one doesn't include param names, which is important for understanding what the callback does IMO.